src/components/common/ShowMoreLess/ShowMoreLess.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './_ShowMoreLess.scss';
+import Container from '../Typography/Container';
+
+class ShowMoreLess extends React.Component {
```

Comment:
Rename it to more meaningful generic name:
```suggestion
class ExpandableText extends React.Component {
```

Reply:
done